test/integrations/destinations/campaign_manager/dataDelivery/oauth.ts

```diff
@@ -326,14 +326,14 @@ export const v1oauthScenarios: ProxyV1TestData[] = [
     },
     output: {
       response: {
-        status: 500,
+        status: 401,
```

Comment:
will this be retried or aborted by rudder server and is that expected

Reply:
authcategory is `REFRESH_TOKEN` , so retired

Reply:
so 4xx status will be handled properly right in rudder server

Reply:
@aashishmalik did you test this behaviour with server. Lets test this scenario buy running transformer and server and running debug-mode on transformer and changing the response in proxy

Reply:
For refresh ideally we should send 500

Reply:
updated in linear, server did refresh call